datastore/dsmodels/fetch_test.go

```diff
@@ -0,0 +1,205 @@
+package dsmodels_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/OpenSlides/openslides-go/datastore/dsmock"
+	"github.com/OpenSlides/openslides-go/datastore/dsmodels"
+)
+
+func TestRequestSingleModel(t *testing.T) {
+	ds := dsmodels.New(dsmock.Stub(dsmock.YAMLData(`---
+	topic/1/sequential_number: 1
+	topic/1/title: foo
+	topic/1/meeting_id: 1
+	topic/1/agenda_item_id: 1
+	topic/1/list_of_speakers_id: 1
+	`)))
+
+	_, err := ds.Topic(1).First(context.Background())
```

Comment:
Go 1.24 introduced [T.Context](https://pkg.go.dev/testing@master#T.Context). I think it will be a good practice to use it in tests.